controllers/component_build_controller.go

```diff
@@ -75,35 +75,78 @@ const (
 )
 
 var (
-	simpleBuildPipelineCreationTimeMetric       prometheus.Histogram
-	pipelinesAsCodeComponentProvisionTimeMetric prometheus.Histogram
+	initialComponentCreationBuildOrPaCProvisionTimeMetric prometheus.Histogram
+	simpleBuildPipelineCreationTimeMetric                 prometheus.Histogram
+	pipelinesAsCodeComponentProvisionTimeMetric           prometheus.Histogram
+	pipelinesAsCodeComponentUnconfigureTimeMetric         prometheus.Histogram
+	pipelinesAsCodeComponentTriggerTimeMetric             prometheus.Histogram
+	componentTimesForMetrics                                  = map[string]componentMetricsInfo{}
 )
 
+type componentMetricsInfo struct {
+	startTimestamp time.Time
+	requestType    string
```

Comment:
By reading through the code, I know that `requestType` is the `requestAction`. How about use same name or give it a short description?

Reply:
sure